lms/djangoapps/certificates/models.py

```diff
@@ -430,6 +428,19 @@ def mark_unverified(self):
             status=self.status,
         )
 
+        if previous_certificate_status == CertificateStatuses.downloadable:
+            # imported here to avoid a circular import issue
```

Comment:
I'd like to avoid importing this here, if possible. 

Could we refactor things to require _get_certificate_url_ to always be provided a user certificate, then _certificate.utils_ wouldn't need to import _GeneratedCertificate_?

Reply:
Sure! I'll take a look and plan on including that update as part of my other changes.

Reply:
I took a look at this and it looked a bit more complicated than originally expected. I created a task to address this concern later (MICROBA-1255).